kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/system/SystemService.java

```diff
@@ -46,6 +46,10 @@
     public static final String KEY_KURA_NAME = "kura.name";
     public static final String KEY_KURA_VERSION = "kura.version";
     /**
+     * @since
```

Comment:
I believe a version needs to be set here

Reply:
done